src/gui/torrentcontentmodelfolder.h

```diff
@@ -64,4 +66,5 @@ class TorrentContentModelFolder final : public TorrentContentModelItem
 
 private:
     QVector<TorrentContentModelItem*> m_childItems;
+    QHash<QString, TorrentContentModelItem *> m_folderMap;
```

Comment:
If you store only folders why not use `TorrentContentModelFolder` instead? From the other hand why not map all the items?